src/main/java/com/questhelper/quests/holygrail/HolyGrail.java

```diff
@@ -0,0 +1,362 @@
+	hasGrailBell, inGrailBellRingLocation, inFisherKingCastle1BottomFloor, inFisherKingCastle1SecondFloor, hasFeather, inFisherKingRealm, inFisherKingCastle2BottomFloor, inFisherKingCastle2SecondFloor, inFisherKingCastle2ThirdFloor,
+	hasGrail;
+
+	QuestStep talkToKingArthur1, talkToMerlin, goUpStairsCamelot, openMerlinDoor, goToEntrana, talkToHighPriest, goToGalahad, talkToGalahad, goToDraynorManor, enterDraynorManor, goUpStairsDraynor1,
+	goUpStairsDraynor2, openWhistleDoor, takeWhistles, goToTeleportLocation1, blowWhistle1, attackTitan, talkToFisherman, pickupBell, ringBell, goUpStairsBrokenCastle, talkToFisherKing, goToCamelot,
+	talkToKingArthur2, openSack, goToTeleportLocation2, blowWhistle2, openFisherKingCastleDoor,  goUpNewCastleStairs, goUpNewCastleLadder, takeGrail, talkToKingArthur3;
+
+	ConditionalStep findFisherKing;
+
+	Zone camelotGround, camelotUpstairsZone1, camelotUpstairsZone2, merlinRoom, entranaBoat, entranaIsland, galahadHouse, draynorManorFront, draynorManorBottomFloor, draynorManorSecondFloor,
+	draynorManorTopFloor, magicWhistleRoom, teleportLocation, fisherKingRealmEntrance, fisherKingRealmAfterTitan1, fisherKingRealmAfterTitan2, fisherKingRealmAfterTitan3, grailBellRingLocation,
+	fisherKingRealmCastle1BottomFloor, fisherKingRealmCastle1SecondFloor, fisherKingRealm, fisherKingRealmCastle2BottomFloor, fisherKingRealmCastle2SecondFloor, fisherKingRealmCastle2ThirdFloor;
+
+	@Override
+	public Map<Integer, QuestStep> loadSteps()
+	{
+		loadZones();
+		setupItemRequirements();
+		setupConditions();
+		setupSteps();
+		Map<Integer, QuestStep> steps = new HashMap<>();
+
+		steps.put(0, talkToKingArthur1);
+
+		ConditionalStep findingMerlin = new ConditionalStep(this, goUpStairsCamelot);
+		findingMerlin.addStep(new Conditions(inMerlinRoom, merlinNearby), talkToMerlin);
+		findingMerlin.addStep(inCamelotUpstairs, openMerlinDoor);
+
+		steps.put(2, findingMerlin);
+
+		ConditionalStep findHighPriest = new ConditionalStep(this, goToEntrana);
+		findHighPriest.addStep(onEntrana, talkToHighPriest);
+
+		steps.put(3, findHighPriest);
+
+		findFisherKing = new ConditionalStep(this, goToGalahad);
+		findFisherKing.addStep(inFisherKingCastle1SecondFloor, talkToFisherKing);
+		findFisherKing.addStep(inFisherKingCastle1BottomFloor, goUpStairsBrokenCastle);
+		findFisherKing.addStep(new Conditions(hasGrailBell, inFisherKingRealmAfterTitan), ringBell);
+		findFisherKing.addStep(talkedToFisherman, pickupBell);
+		findFisherKing.addStep(inFisherKingRealmAfterTitan, talkToFisherman);
+		findFisherKing.addStep(new Conditions(hasExcalibur, titanNearby), attackTitan);
+		findFisherKing.addStep(new Conditions(hasTwoWhistles, inTeleportLocation, hasExcalibur), blowWhistle1);
+		findFisherKing.addStep(new Conditions(hasTwoWhistles, hasExcalibur), goToTeleportLocation1);
```

Comment:
This looks like a player will get stuck here if they haven't taken excalabur with them to Draynor Manor to get the whistles. If we're wanting require the player to get Excalabur before this prompt, probably worth us adding in a 'go get excalabur from the bank' step between this and the below, like:

`findFisherKing.addStep(new Conditions(hasTwoWhistles), goGetExcalabur);`